docs-v2/integrations/all/unauthenticated.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Unauthenticated
+sidebarTitle: Unauthenticated
+---
+
+API configurations: [`unauthenticated`](https://nango.dev/providers.yaml)
+
+This is a testing provider, it is not doing anything and not recommended to use in production.
```

Comment:
I don't know if I would say it is not recommended to use in production. It has use cases.

I would rather say,

> This provider doesn't require credentials and no authentication flow is performed when creating connections.

...or something like that.

Reply:
Yeah makes sense I wasn't sure how we wanted to advertise this one. Changing 👌🏻